open_event/views/admin/models_views/roles.py

```diff
@@ -0,0 +1,28 @@
+from flask_admin import expose
+from flask_admin.contrib.sqla import ModelView
+from flask import request, url_for, redirect
+from ....helpers.data import DataManager
+from ....helpers.data_getter import DataGetter
+
+
+class RoleView(ModelView):
+
+    @expose('/new/', methods=('GET', 'POST'))
+    def create_view(self, event_id):
+        if request.method == 'POST':
+            DataManager.add_role_to_event(request.form, event_id)
+        return redirect(url_for('event.details_view', event_id=event_id))
+
+    @expose('/<uer_id>/delete/', methods=('GET',))
+    def delete_view(self, event_id, uer_id):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2686438544/issues/source?bid=3233864&fileBranchId=3324174#l17)